cogs/vxtwitconverter/core.py

```diff
@@ -1,15 +1,20 @@
 import logging
 import os
 
-from redbot.core import data_manager
+from redbot.core import Config, data_manager
 from redbot.core.bot import Red
 
+from .constants import DEFAULT_GUILD
```

Comment:
There is no `constants.py` file in your commit 🤔

Reply:
omg i forgot to git add